examples/demos/boot_from_SAN_and_server_profile_and_volume_migration/README.md

```diff
@@ -0,0 +1,17 @@
+# Demo
+## Server profile with boot from SAN and server profile & volume migration
+
+This demo is composed by 5 steps, each one associated with a playbook, that should be executed in the specified order: 
+
+- `step_1_create_server_profile_boot_from_SAN_and_assign_to_bay1.yml`: This playbook exemplifies how to create a server profile with boot from SAN (3PAR) and without using a template. On task #1, the server profile is created (if not already present) and assigned to server hardware in bay 1. You can check all parameters, including those related to the server hardware and those related to the storage as SAN connections and volume attachment. The volume must be ready to use. On task #2, the server hardware is powered on.
```

Comment:
I'd suggest swapping out this style of writing:
'On task #1,' >>>

For this one:
It performs 2 tasks:
1.  the server profile is created (...)
2. The server hardware is powered on.

This is specially important since the #1 and #2 in this will actually automatically link to issues which is not the intended purpose.

Reply:
#1 and #2 were not converted to links, but I changed to the suggested format.